falcon_kit/bash.py

```diff
@@ -294,6 +294,8 @@ def script_run_consensus(config, db_fn, las_fn, out_file_bfn):
     params.update(locals())
     if config["falcon_sense_skip_contained"]:
         run_consensus = """LA4Falcon -H$CUTOFF -fso {db_fn} {las_fn} | """
+    elif config["falcon_greedy"]:
```

Comment:
Can @ylipacbio comment WRT the name? This option is exposed to pbsmrtpipe, so I don't think it has to be in the convention of "falcon_sense_skip_contained", but I do agree that it could have a better name like "falcon_aggressively_contained" or something.

The exclusiveness LGTM.